src/main/java/unrc/dose/Proposition.java

```diff
@@ -0,0 +1,7 @@
+package unrc.dose;
+
+import org.javalite.activejdbc.Model;
+
```

Comment:
Could you add the schema information of this class?

Reply:
ready !